embeds/watch-me-date/index.html

```diff
@@ -0,0 +1,15 @@
+<a class="watch-me-date__anchor" href="http://www.theguardian.com/lifeandstyle/ng-interactive/2015/feb/12/watch-me-date"></a>
+<div class="watch-me-date__content">
+    <h1 class="watch-me-date__title">Watch Me Date</h1>
+    <h2 class="watch-me-date__subtitle">
+        <span class="watch-me-date__subtitle-line--1">One couple, </span>
+        <span class="watch-me-date__subtitle-line--2">two cameras, </span>
+        <span class="watch-me-date__subtitle-line--3">what happens next?</span>
+    </h2>
+    <div class="watch-me-date__button-lockup">
+        <svg class="watch-me-date__marque" xmlns="http://www.w3.org/2000/svg" width="66" height="59" viewBox="0 0 36 32"><g fill="#fff"><path d="M10.477.14v.002V.14zM8.79 15.997c.467-.492.935-.936 1.405-1.336.468-.396.89-.713 1.265-.95l-.14-.07c-1.22-.7-2.203-1.64-2.953-2.81-.75-1.17-1.125-2.577-1.125-4.22 0-1.406.282-2.646.845-3.726C8.65 1.808 9.444.89 10.477.142 8.928.844 7.52 1.76 6.256 2.884 4.994 4.01 3.89 5.274 2.954 6.68c-.94 1.406-1.665 2.943-2.18 4.606C.257 12.95 0 14.696 0 16.523c0 2.486.467 4.797 1.405 6.924.937 2.135 2.203 4.02 3.797 5.66v-.28c0-.796.178-1.476.528-2.038s.808-1.03 1.37-1.407c.562-.373 1.195-.68 1.898-.913.703-.233 1.406-.423 2.11-.562-1.36-.282-2.353-.845-2.987-1.688-.633-.844-.95-1.758-.95-2.742 0-.654.152-1.275.46-1.862.302-.587.69-1.127 1.16-1.62zM21.445 25.383h-6.68c-1.125 0-1.888.316-2.285.948-.4.635-.597 1.28-.597 1.936 0 .89.456 1.674 1.37 2.354.914.68 2.542 1.02 4.887 1.02 2.672 0 4.465-.327 5.38-.983s1.37-1.477 1.37-2.46c0-.845-.306-1.523-.916-2.04-.61-.516-1.45-.774-2.53-.774zM36 16.52c0-1.873-.272-3.655-.81-5.343-.54-1.687-1.29-3.245-2.25-4.675-.96-1.43-2.108-2.695-3.444-3.797C28.16 1.605 26.693.7 25.1-.002c1.077.657 1.92 1.537 2.533 2.637.608 1.103.937 2.403.985 3.903 0 2.72-.975 4.816-2.917 6.293-1.946 1.478-4.512 2.216-7.7 2.216-.797 0-1.51-.023-2.144-.07-.633-.046-1.278-.187-1.933-.422-.33.235-.6.517-.81.845-.21.327-.316.656-.316.983 0 .422.175.8.527 1.125.35.328.76.492 1.23.492h8.86c2.765 0 4.813.56 6.15 1.686 1.333 1.123 2.003 2.906 2.003 5.344 0 1.642-.327 3.07-.983 4.29 1.642-1.64 2.953-3.552 3.938-5.73.986-2.18 1.478-4.537 1.478-7.067v-.002zM17.916 11.21s4.314-3.89 4.314-6.42-3.167-2.698-4.314-.658c-1.147-2.04-4.314-1.87-4.314.66s4.314 6.417 4.314 6.417z"/></g></svg>
```

Comment:
need to remove the `xmlns="http://www.w3.org/2000/svg"` othewise IE8 will freakout. also you know IE8/android 2.* won't see the image? that ok?